Make a separate factory for parsing the input to the LLM task (#81)
explosion/spacy-llm: README.md

need to implement two functions: `generate_prompts` that takes a list of spaCy [`Doc`](https://spacy.io/api/doc) objects and transforms
them into a list of prompts, and `parse_responses` that transforms the LLM outputs into annotations on the [`Doc`](https://spacy.io/api/doc), e.g. entity spans, text categories and more.

> To register your custom task with spaCy, decorate a factory function using the `spacy_llm.registry.llm_tasks`

Reviewer: ```suggestion
To register your custom task with spaCy, decorate a factory function in `spacy_llm.registry.llm_tasks`
```
